lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -467,9 +486,9 @@ def calculate_wet_bulb_temperature(self, temperature, relative_humidity,
 
             # Recalculate the saturation mixing ratio
             saturation_mixing_ratio = self._calculate_mixing_ratio(
-                wbt, pressure)
+                wbt_data, pressure)
 
-        return wbt
+        return wbt_data
 
     def process(self, temperature, relative_humidity, pressure):
```

Comment:
As far as I can tell this still uses calculate_wet_bulb_temperature in order to calculate the wet bulb temperature. However you have now changed this so it doesn't do what it originally did (there is no iteration within this calculation now?). Do you mean to use _calculate_wbt within process now to calculate the wet bulb temperature?

Reply:
`calculate_wet_bulb_temperature` calls `_calculate_wbt`
```
wbt_data = self._calculate_wbt(
        precision, pressure.data, relative_humidity.data, temperature.data)
```